README.md

```diff
@@ -24,7 +24,9 @@ Run the unit and integration tests with code coverage reporting using `./run_all
 
 ## Service responsibilities
 
-This is the front-end for the functionality to create test Individuals and Organisations that can subsequently be used for testing of API microservices in the External Test environment.
+This is the front-end for the functionality to create test Individuals and Organisations that can subsequently be used for testing of API microservices in the External Test environment. 
+The `api-platform-test-user-frontend` runs in the primary environment (ie Production and QA) but is configured to talk, via a proxy, to the `api-platform-test-user` backend service - which handles the work of actually creating the test users in a Mongo database - in the sandbox environment (ie External Test and Development, respectively). This is to allow testing to take place in External Test and keep Production systems free of test data.
```

Comment:
Might be worth adding that the backend service is called as an API Platform API, as that also caught us out a little bit.